documentation/specs/TabPanels.md

```diff
@@ -0,0 +1,163 @@
+# `TabPanels` Component Specification
+
+## Overview
+
+A `TabPanels` is a set of styled links that let users switch between multiple views within a related context under the same URL.
```

Comment:
This says "a set of styled links", but that seems confusing with the below line of "should not be links that navigate the user to a new URL". Curious if they are still actually links (query params?) or something else?

Reply:
agreed that's confusing. updated